algorithms/schema-free/greedy_graph_coloring.gsql

```diff
@@ -0,0 +1,78 @@
+CREATE QUERY greedy_graph_coloring(SET<STRING> v_type,SET<STRING> e_type,BOOL print_color_count = TRUE,BOOL display = TRUE, STRING file_path = ""){
```

Comment:
Should we name it graph_coloring_greedy instead?
The general naming convention is to state the general algorithm first, and then add on modifiers.